### DISCUSSION TITLE
Remember position on last page when going back

### DISCUSSION THREAD

**Discussion by KillerParticles** [ciderapp/Cider#685] [Feature Request]
For example, when you browse your Listen now page, you scroll down half way and click on a Album to check it out, when you go back, you expect to be in half way down right before you clicked on the album right? Well you don't, and you get sent back to the top of the page, meaning you have to scroll all the way back down, and have to do this over and over again. This also happens in the search bar, settings menu and more.

The feature is for Cider to remember your position on the page so when you go back, you are on the same position on the page as you were before clicking on a album or song.

**Comment by JohnLennon3** [comment]
I noticed that this is the case since the previous page scrolls to the top before going to the page you clicked on. The last scroll position is therefore set to the top of the page, and that is what is restored when history.back is called.

**Comment by nxvh** [comment]
i made an account just to say how much this sucks, especially when trying to make a playlist

**Reply by cryptofyre** [reply]
We've fixed this in a newer release that will be coming to the Microsoft Store soon.
